### PR TITLE
add styling for queries which are run inside a transaction

### PR DESCRIPTION
**Pull request by LordSimal**
image

Feel free to suggest a better background color to group queries inside a transaction

### CODE REVIEW

File: templates/element/sql_log_panel.php [hunk 1]
```diff
@@ -91,6 +91,12 @@
                             <td><?= h($query['rows']) ?></td>
                             <td><?= h($query['took']) ?></td>
                         </tr>
+                        <?php if($query['isCommitOrRollback']): ?>
+                            <tr>
+                                <td colspan="3" class="commit-or-rollback">
+                                </td>
+                            </tr>
+                        <?php endif; ?>
```
Comment (LordSimal) [inline templates/element/sql_log_panel.php:99]: This was added to add space between the queries.

File: templates/element/sql_log_panel.php [hunk 2]
```diff
@@ -91,6 +91,12 @@
                             <td><?= h($query['rows']) ?></td>
                             <td><?= h($query['took']) ?></td>
                         </tr>
+                        <?php if($query['isCommitOrRollback']): ?>
```
Comment (ADmad) [inline templates/element/sql_log_panel.php]:
```suggestion
                        <?php if ($query['isCommitOrRollback']): ?>
```

File: webroot/css/style.css [hunk 3]
```diff
@@ -605,6 +607,10 @@ strong {
     box-shadow: 0 2px 0 var(--routes-btn-active-border);
 }
 
+.c-sql-log-panel__entry .in-transaction {
+    background: var(--sql-log-transaction);
```
Comment (ADmad) [inline webroot/css/style.css]:
```suggestion
    background: var(--cake-light-gray);
```

**Comment by LordSimal** [comment]
This is the new color from ADmad
image

File: webroot/css/style.css [hunk 4]
```diff
@@ -60,6 +60,8 @@
     --packages-warning: #ffcc00;
     --packages-link: #000;
 
+    --sql-log-transaction: #d8d8d8;
```
Comment (ADmad) [inline webroot/css/style.css]: Not needed anymore

**Comment by othercorey** [comment]
Thanks!

**Pull request merged**